webapp/config/packages/security.yaml

```diff
@@ -33,8 +33,8 @@ security:
         # API does Basic Auth and IP address auth
         api:
             pattern: ^/api
+            context: domjudge
             provider: domjudge_db_provider
-            stateless: true
```

Comment:
Does this have a (measurable) performance penalty because we need to remember the sessions?

Reply:
It does a `session_start` but that shouldn't have any measurable impact.